events/2022-google-summer-of-code/index.markdown

```diff
@@ -35,31 +41,46 @@ MoveIt is planning to participate again in the [Google Summer of Code](https://s
   * What are the next positions of the other currently active robot arms?
 
   Much of the preliminary work is done in [this PR](https://github.com/ros-planning/moveit/pull/2810).
-* Project size: large
+* Expected outcome: MoveIt is capable of supporting simultaneous motion with two arms.
+* Project size: large (350 hours)
+* Difficulty: hard
+* Preferred skills: several years experience in (ROS, MoveIt, C++)
 * Mentor: Henning Kayser
 
 ### Add ability to attach/detach end-effector links
 
 * Add functionality to MoveIt to programmatically attach and detach grippers from a manipulator. See [this issue](https://github.com/ros-planning/moveit2/issues/208) for a more detailed discussion.
-* Project size: large
+* Expected outcome: End-effector tooling can be swapped programmatically.
+* Project size: large (350 hours)
+* Difficulty: hard
+* Preferred skills: several years experience in (ROS, MoveIt, C++)
 * Mentor: Andy Zelenak
 
 ### Tutorial for multiple robot arms
 
 * While there are some ROS Answers posts and examples floating around, there is no definitive resource on how to set up multiple manipulators with MoveIt (and especially MoveIt2). The goal of this project is to write a tutorial that should become the reference.
-* Project size: medium
+* Expected outcome: A ROS beginner can read the tutorial and set up a ros2_control / MoveIt pipeline without additional help.
+* Project size: medium (175 hours)
+* Difficulty: easy
+* Preferred skills: at least a year of experience with (ROS, MoveIt)
```

Comment:
```suggestion
* Preferred skills: Technical writing, ROS, and MoveIt
```

Reply:
OK. This seems pretty nit-picky, but whatever.